pkgs/top-level/python-packages.nix

```
@@ -958,6 +958,8 @@ in {
 
   pydocumentdb = callPackage ../development/python-modules/pydocumentdb { };
 
+  PyDrive = callPackage ../development/python-modules/PyDrive { };
```

```suggestion
  pydrive = callPackage ../development/python-modules/pydrive { };
```

I believe attributes in `python-packages.nix` should be lowercase.
Think we want to normalize `pname` that way also.

---

Do you have a reference for that? I only see [15.17.1.2.1. Packaging a library](https://nixos.org/nixpkgs/manual/#packaging-a-library), which says “One of the arguments is the name of the package, which consists of a basename (generally following the name on PyPi)”

---

https://nixos.org/nixpkgs/manual/#contributing `attribute names` and the one about the `name` attribute is much harder to fine. (or maybe the latter isn't documented yet).

---

Done thanks.